frontend/src/Setting/Admin/UserAdd/UserAddPane.tsx

```diff
@@ -61,12 +62,20 @@ function UserAddPane() {
     });
   }
 
-  async function addUser(data: TUserAddFormInput) {
+  async function handleAddMember(data: TUserAddFormInput) {
     const { userId, userName, attribute } = data;
 
     if (!(await Backend.addMember(userId, userName, attribute))) {
-      throw new Error();
+      throw new Error("addMember: failed");
     }
+
+    const dispatch = useMembersDispatch();
+    dispatch({
+      type: "added",
+      id: userId,
+      name: userName,
+      attribute: attribute,
+    });
```

Comment:
メンバーを新たに追加した時，追加に成功していてもexceptionが発生し，ユーザの追加に失敗した旨を伝えるトーストが表示されます．
これは，フックをトップレベル以外の場所で呼び出しているためだそうです．
https://qiita.com/tatsumin0206/items/4e1076e2deedf20a9485

`const dispatch = useMembersDispatch()`を`const toast = useToast();`の下辺りに移動させてください．

```suggestion
    dispatch({
      type: "added",
      id: userId,
      name: userName,
      attribute: attribute,
    });
```